src/test.js

```diff
@@ -344,7 +344,7 @@ Test.prototype = {
 			// generating stack trace is expensive, so using a getter will help defer this until we need it
 			get source() { return test.stack; }
 		} ).then( function() {
-			if ( module.testsRun === numberOfTests( module ) ) {
+			if ( ( module.testsRun + module.testsNotRan ) === numberOfTests( module ) ) {
```

Comment:
do we also need to check for parent module as well (#353)?
`while ( parent && parent.testsRun + parent.testNotRan === numberOfTests( parent ))`

Reply:
@choheekim Do you have a concrete example to help me understand the question?
 I am not sure if I can visualize the problem yet

Reply:
Talked to @choheekim offline, she pointed out the nested modules scenario, which pointed out something missing in the solution. Revised it to include this usecase.